test-storybooks/portable-stories-kitchen-sink/nextjs/package.json

```diff
@@ -6,7 +6,8 @@
     "build": "next build",
     "cypress": "echo 'No cypress setup yet'",
     "dev": "next dev",
-    "jest": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "jest": "echo 'No jest setup yet'",
+    "jest-": "node node_modules/jest/bin/jest.js",
```

Comment:
What's the reason for disabling jest runs for the `portable-stories-kitchen-sink/nextjs` project?

Reply:
portable stories in jest do not work, we knew this going in to the project.